archinstall/lib/installer.py

```diff
@@ -34,7 +34,8 @@ class Installer():
 	:type hostname: str, optional
 
 	"""
-	def __init__(self, target, *, base_packages='base base-devel linux linux-firmware efibootmgr'):
+	def __init__(self, target, *, base_packages=' base base-devel linux-firmware efibootmgr', kernel='linux'):
```

Comment:
make it be kernels (plural) and accept an array. For instance kernels=['linux', 'linux-lts'[